articles/azure-monitor/agents/azure-monitor-agent-troubleshoot-windows-vm.md

```diff
@@ -28,7 +28,11 @@ Follow the steps below to troubleshoot the latest version of the Azure Monitor a
 	4. If not, check if you see any errors in extension logs located at `C:\WindowsAzure\Logs\Plugins\Microsoft.Azure.Monitor.AzureMonitorWindowsAgent` on your machine   
 	5. If none of the above helps, [file a ticket](#file-a-ticket) with **Summary** as 'AMA extension fails to install or provision' and **Problem type** as 'I need help with Azure Monitor Windows Agent'.    
 	
-3. **Verify that the agent is running**:  
+3. **Verify that the agent is running**: 
+
+	> [!NOTE]
+	> The prerequisites mentioned at [Azure Monitor Agent - Prerequisites](azure-monitor-agent-manage.md?tabs=azure-portal#prerequisites) must be met prior proceeding with the below steps. Without a managed identity assigned to the VM, the process **MonAgentCore.exe** will fail to start.
```

Comment:
This is already mentioned as step no. 1 in this list. User is expected to follow step 1 before executing step 3.